src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/InvokeRestMethodCommand.Common.cs

```diff
@@ -60,6 +60,13 @@ public int MaximumFollowRelLink
             set { base._maximumFollowRelLink = value; }
         }
 
+        /// <summary>
+        /// Gets or sets the ResponseHeadersVariable property.
+        /// </summary>
+        [Parameter]
+        [Alias("RHV")]
+        public string ResponseHeadersVariable { get; set; }
```

Comment:
I chose `Headers` because we are using the `-Headers` parameter as well as `WebResponseObject.Headers` (which this provides parity for). I can change it, but it would be the odd-man-out.

Reply:
I'm fine with it